### DISCUSSION TITLE
Equivalenece  classe file

### DISCUSSION THREAD

**Discussion by Ray6283** [COMBINE-lab/salmon#883] [General]
From https://salmon.readthedocs.io/en/latest/file_formats.html#quantification-file ,
it says "If salmon was run with the --dumpEq option, then a file called eq_classes.txt will exist in the auxiliary directory"
my question is how do we run with --dumpEq option?
I am following steps described in https://combine-lab.github.io/salmon/getting_started/
and could get the results (the quants file) , but don't know where to put --dumpEq to get the Equivalence class file.
Thanks.

**Comment by rob-p** [comment]
duplicate of https://github.com/COMBINE-lab/salmon/discussions/882#discussioncomment-7256507
